docs/userGuide/components/advanced.md

```diff
@@ -71,6 +71,33 @@ Using the normal syntax, you are only able to use markdown formatting on heading
 </tip-box>
 <br>
 
+**Box Slot Options:**
+
+Slot name | Default class |
+--- | --- |
+icon | (depends on box's `type` attribute) |
+
+*Example 1*: Override the default icon for a certain type of box.
+
+```html
+<box type="info">
+    <span slot="icon" class="text-danger"><md>:fas-home:</md></span>
+    info
+</box>
+```
+
+<box type="info">
+    <span slot="icon" class="text-danger"><md>:fas-home:</md></span>
+    info
+</box>
+
+*Example 2*: Use pictures (or even gifs) as icon for a box.
+
+<box type="info">
+    <img slot="icon" src="https://icons8.com/vue-static/landings/animated-icons/icons/cloud/cloud.gif"></img>
+    some very useful info
+</box>
+
```

Comment:
Can post a screenshot? I can't see the final look from the PR preview.

Reply:
![63a0b9f4-8658-4f6d-9077-d8102c6152b2](https://user-images.githubusercontent.com/28982486/77131720-a17ab180-6a97-11ea-8a7c-231c3c771089.gif)

Reply:
For the animated gif, can find one without background (or make the box background white) so that it blends in nicely? We should try to make the examples as good looking as possible.

Can also add an example using a thumbnail as an icon (i.e., using MarkBind thumbnails feature), possibly a thumbnail of a letter or a number?